Cooper/ClientApp/src/app/services/user.service.ts

```diff
@@ -10,78 +10,38 @@ import { AuthService, FacebookLoginProvider, GoogleLoginProvider } from 'ng-dyna
 export class UserService {
 
   constructor(private fb: FormBuilder, private router: Router, private http: HttpClient, private socialAuthService: AuthService) { }
+
   public user: any;
   readonly registrationUrl = '/registration';
   readonly authUrl = '/auth/login';
   readonly getProvider = {
-    "google": GoogleLoginProvider.PROVIDER_ID,
-    "facebook": FacebookLoginProvider.PROVIDER_ID
-  }
-
-  formModel = this.fb.group({
-    UserName: ['', Validators.required],
-    Email: ['', [Validators.email, Validators.required]],
-
-    Passwords: this.fb.group({
-      Password: ['', [Validators.required, Validators.minLength(4)]],
-      ConfirmPassword: ['', Validators.required]
-    }, { validator: this.comparePasswords })
-
-  });
+    'google': GoogleLoginProvider.PROVIDER_ID,
+    'facebook': FacebookLoginProvider.PROVIDER_ID
+  };
 
   public CheckAuthentification(): void {
```

Comment:
decide low/upper letter in start of function's name

Reply:
done